src/utils/businessHours.ts

```diff
@@ -169,7 +169,8 @@ export async function getNextAvailableBusinessHourWindow(
   org?: string
 ): Promise<BusinessHourWindow> {
   let offices: string[] = [];
-  if (repo && org && GH_ORGS.get(org).repos.withoutRouting.includes(repo)) {
+  // TODO(team-ospo/issues#200): Add codecov support
```

Comment:
nit: I think github style is easier to read. The "issues" bit is redundant and will appear in every such comment. `[ORG]/[REPO]#[ISSUE_NUM]` is all relevant information, and universally used across GitHub. So `getsentry/team-ospo#200` in this case.